data/map planets.txt

```diff
@@ -136,6 +137,14 @@ planet "Alta Hai"
 	bribe 0
 	security 0.9
 
+planet Alvorada
+	attributes ccor military station
+	landscape land/station8
+	description `Formerly "Fortaleza da Alvorada," Alvorada was hollowed out centuries ago to make an extensive network of bunker complexes that used to house the Politburo of the CCOR, and the extensive computer system they built that formulated their systematic plans. Rumor has it that, in the deepest reaches of the complex where no pirates have yet been able to breach, they're still there, having adapted the system to instead direct the guerrillas in the reestablishment of their homeland.`
```

Comment:
What is "systematic plans" supposed to mean, by the way?

```suggestion
	description `Formerly "Fortaleza da Alvorada," Alvorada was hollowed out centuries ago to make an extensive network of bunker complexes that used to house the Politburo of the CCOR and the extensive computer system they built that formulated their systematic plans. Rumor has it that, in the deepest reaches of the complex where no pirates have yet been able to breach, they're still there, having adapted the system to instead direct the guerrillas in the reestablishment of their homeland.`
```

Reply:
5-year plans, I imagine

Reply:
Definitions from [Oxford Languages](https://languages.oup.com/google-dictionary-en) · [Learn more](https://support.google.com/websearch/answer/10106608?hl=en)
sys·tem·at·ic
/ˌsistəˈmadik/
adjective
done or acting according to a fixed plan or system; methodical.
"a systematic search of the whole city"

Reply:
From Merriam-

Systemic describes what relates to or affects an entire system. For example, a systemic disease affects the entire body or organism, and systemic changes to an organization have an impact on the entire organization, including its most basic operations.

So, I would assume plans in regards to the CCOR as a whole, or at the very least Alvorada.

Reply:
That's the definition for `systemic` rather than `systematic`, though.

Reply:
> 5-year plans, I imagine

Yep, this is the intention. Didn't want to specify an interval but this is meant to imply CCOR's economy was planned.

Reply:
```suggestion
	description `Formerly "Fortaleza da Alvorada," Alvorada was hollowed out centuries ago to make an extensive network of bunker complexes that used to house the Politburo of the CCOR and the extensive computer system they built that administered their centrally-planned economy. Rumor has it that, in the deepest reaches of the complex where no pirates have yet been able to breach, they're still there, having adapted the system to instead direct the guerrillas in the reestablishment of their homeland.`
```
How about this?

Reply:
Systemic is pretty close, though perhaps we move away from either word and reword it to something else.